internal/db/schema/migrations/postgres/10/03_vault_credential.up.sql

```diff
@@ -315,51 +320,55 @@ begin;
-            store.ca_cert           as ca_cert,
-            store.tls_server_name   as tls_server_name,
-            store.tls_skip_verify   as tls_skip_verify,
-            store.public_id         as store_id,
-            token.token_hmac        as token_hmac,
-            token.token             as ct_token, -- encrypted
-            token.create_time       as token_create_time,
-            token.update_time       as token_update_time,
-            token.last_renewal_time as token_last_renewal_time,
-            token.expiration_time   as token_expiration_time,
-            token.key_id            as token_key_id,
-            token.status            as token_status,
-            cert.certificate        as client_cert,
-            cert.certificate_key    as ct_client_key, -- encrypted
+     select store.public_id           as public_id,
+            store.scope_id            as scope_id,
+            store.name                as name,
+            store.description         as description,
+            store.create_time         as create_time,
+            store.update_time         as update_time,
+            store.version             as version,
+            store.vault_address       as vault_address,
+            store.namespace           as namespace,
+            store.ca_cert             as ca_cert,
+            store.tls_server_name     as tls_server_name,
+            store.tls_skip_verify     as tls_skip_verify,
+            store.public_id           as store_id,
+            token.token_hmac          as token_hmac,
+            token.token               as ct_token, -- encrypted
+            token.create_time         as token_create_time,
+            token.update_time         as token_update_time,
+            token.last_renewal_time   as token_last_renewal_time,
+            token.expiration_time     as token_expiration_time,
+            token.renewal_time        as token_renewal_time,
+            token.key_id              as token_key_id,
+            token.status              as token_status,
+            cert.certificate          as client_cert,
+            cert.certificate_key      as ct_client_key, -- encrypted
             cert.certificate_key_hmac as client_cert_key_hmac,
-            cert.key_id             as client_key_id
+            cert.key_id               as client_key_id
        from credential_vault_store store
-  left join current_tokens token
+  left join active_tokens token
          on store.public_id = token.store_id
   left join credential_vault_client_certificate cert
          on store.public_id = cert.store_id;
   comment on view credential_vault_store_client_private is
     'credential_vault_store_client_private is a view where each row contains a credential store and the credential store''s data needed to connect to Vault. '
+    'The view returns a separate row for each current and maintaining token, maintaining tokens should only be used for token/credential renewal and revocation.'
```

Comment:
```suggestion
    'The view returns a separate row for each current and maintaining token, maintaining tokens should only be used for token/credential renewal and revocation. '
```
Postgres concatenates all of the lines in the comment so each line (except for the last one) needs to end with a space.